lego-webapp/pages/bdb/company-interest/Translations.ts

```diff
@@ -236,6 +232,16 @@ export const FORM_LABELS = {
     norwegian: 'Har dere kontorer i Trondheim egnet for besøk?',
     english: 'Do you have offices in Trondheim suited for visiting?',
   },
+  wantsThursdayEvent: {
+    norwegian: 'Ønsker dere arrangement på torsdag?',
+    english: 'Would you like to have the event on Thursday?',
+  },
+  wantsThursdayEventInfo: {
+    norwegian:
+      'Torsdags-arrangementer er av de mest populære blant studentene, og vi ser ofte høyere påmelding og større engasjement disse dagene.',
+    english:
+      'Thursday events are among the most popular with students, and we often see higher attendance and greater engagement on these days.',
```

Comment:
```suggestion
    norwegian:
      'Torsdags-arrangementer er av de mest populære blant studentene, og vi ser ofte høyere påmelding og større engasjement på torsdager.',
    english:
      'Thursday events are among the most popular with students, and we often see higher attendance and greater engagement on thursdays.',
```
Or 'this day', 'events on this day'?

Reply:
➕